wiki/People/The_Team/Beatmap_Nominators/Rules/tr.md

```diff
@@ -0,0 +1,70 @@
+- **Eğer herhangi bir şeyden emin değilseniz beatmapi aday göstermeyin.** Bu gibi durumlarda, acele bir karar vermeden önce sevgili aday göstericilerden yardım isteyin. Bu, iyi bilinmeyen oyun modlarını, gri alanda bulunan/derecelendirilemez olabilecek şüpheli hataları, kalite sorunları ya da karmaşık zamanlamaları da içerir.
+- **Akıl çerçevesinde değerlendiremeyeceğiniz beatmapi aday göstermeyin.** Bir beatmapi akıl çerçevesinde değerlendirmek, oynayarak test ederek veya modlama tecrübesiyle, bir beatmapin kalitesini değerlendirebilmenin çekirdeğini oluşturur. Eğer yeteri kadar oyun yeteneğiniz ya da modlama deneyiminiz olmadığı varsayılıyorsa, beatmapi değerlendirme yeteneğinizden şüphe duyabiliriz.
+- **Resmiyette yeterliliğinizin olmadığı oyun modlarındaki beatmapleri aday göstermeyin.** Aday gösterebilmeniz için Beatmap Aday Göstericilerin ilgili kısmının bir üyesi olarak beatmap setinde bulunan oyun modlarından en az birinde aday göstermek için kalifiye olmalısınız.
+- **Beatmapi aday göstermeden önce beatmap tartışmasında hiç bir gönderinin göz ardı edilmediğinden emin olun.** Elverişli bir yanıtı bulunmayan gönderiler yoksayılmış farz edilebilir. Şüphede kaldığınızda, beatmap tartışmasında Mapleme ve Modlama Etik Davranış Kuralları ihlal edilmiş mi diye iki kez kontrol edin ve ona göre davranın.
+
+### Niteliklendirme
+
+- **Bir şarkının benzer versiyonlarının farklı beatmapleri aynı modda içerisinde aynı anda niteliklendirilemez.** Kısaltmalar, uzatmalar, ya da basit aranjmanlar şarkının benzer versiyonlarına dahildir.
+- **Bir şarkının önemli derecede farklı versiyonlarından en fazla 3 tanesi aynı mod içerisinde aynı anda niteliklendirilebilir.** Remiksler, derlemeler, ve karıştırmalar şarkının önemli derecede farklı versiyonlarına dahildir.
+- **Asil üyesi olmadığnız modlardaki beatmapleri, eğer olan bir başkası tarafından aday gösterilmediyse niteliklendirmeyin.** Bu, bir asil aday göstericinin her bir modda sürecin her daim bir parçası olmasını sağlar.
+- **Bir beatmapi mapper ile iletişim kurmadan doğrudan niteliklendirmeyin.** Son aday göstermeden sonra mapperın kendisi bazı değişiklikler yapmış olabilir, dolayısıyla mapper ile irtibat halinde olmadan bir beatmap setini aday göstermek gereksiz diskalifikasyonlara yol açabilir.
+- **Melez beatmap setleri niteliklendirilebilmeleri için her iki modun asil üyeleri tarafından aday gösterilmeli**, aynı zamanda modlama tartışmalarında diğer modların aday göstericilerinin açık onayını kazanmaları gerekir. Bu, sadece asil üyelerin melez setleri aday gösterebileceği anlamına gelir, ve her bir mod için birer asil aday göstericinin (aday göstermek ya da sadece tartışma içerisinde onay vermek olsun fark etmez) sorumluluğu üstleneceğini garantiler. Bu doğrultuda, birden fazla modda uzman olan aday göstericiler birden fazla modda onaylayabilirler.
```

Comment:
I think I elaborated on why `melez` and `niteliklendirme` are used, the former is the most natural form of `hybrid` in Turkish and the latter is simply because `qualified` = `nitelikli` on the website